tests/fixtures/celery.py

```diff
@@ -39,7 +39,11 @@ def celery_pydantic_config() -> CeleryConfiguration:
 
     The config returned will then be used to configure the `celery_app` fixture.
     """
-    return CeleryConfiguration.model_construct(broker_url="memory://")
+    return CeleryConfiguration.model_construct(
+        broker_url="memory://",
+        result_backend="cache+memory://",
+        task_always_eager=True,
```

Comment:
We override task_always_eager here so that tasks that do not return results are completed when .wait() returns.  If we don't do this, tests will fail because when ignore_task=True,  the .wait() invocation will return immediately before the task has been processed.